tests/testthat/test-digest.R

```diff
@@ -0,0 +1,63 @@
+test_that("sha1.rxUi", {
+  mod1 <- function() {
+    ini({
+      KA=2.94E-01
+      CL=1.86E+01
+      V2=4.02E+01
+      Q=1.05E+01
+      V3=2.97E+02
+      Kin=1
+      Kout=1
+      EC50=200
+    })
+    model({
+      C2 <- centr/V2
+      C3 <- peri/V3
+      d/dt(depot) <- -KA*depot
+      d/dt(centr) <- KA*depot - CL*C2 - Q*C2 + Q*C3
+      d/dt(peri)  <- Q*C2 - Q*C3
+      eff(0) <- 1
+      d/dt(eff)   <- Kin - Kout*(1-C2/(EC50+C2))*eff
+    })
+  }
+
+  ui1 <- rxode2(mod1)
+  ui1$mv0$timeId
+
+  # Wait a few seconds so that compilation will occur at a different time
+  Sys.sleep(2)
+
+  mod2 <- function() {
+    ini({
+      KA=0.294
```

Comment:
@mattfidler, I used normal numbers rather than SI notation to try to make the models compile differently, but it didn't work.

Reply:
This is because R handles the numbers and SI vs normal numbers are stored in R exactly the same, so it shouldn't have differences.  This is expected.

Also rxode2 uses normalized model syntax to compare models.  So

```
x^2
```

Is equivalent to

```
x**2
```

Which it should be since it generates the same code :smile: